app/search/template.hbs

```diff
@@ -148,3 +191,118 @@ as |layout|>
         {{/unless}}
     </layout.main>
 </OsfLayout>
+{{!-- OSF Search Help Guide --}}
+{{!-- template-lint-disable no-heading-inside-button --}}
+    <EmberPopover
+        data-test-search-help-1
+        @targetId='allItems'
+        @tooltipClass={{local-class 'search-help-tutorial-1'}}
+        @innerClass={{'search-help-tutorial-inner'}}
+        @isShown={{this.showTooltip1}}
+        @event='none'
+        @side='bottom'
+        @spacing={{20}}
+    >
+        <aside local-class='help-content'>
+            <h2 local-class='search-help-header' data-test-help-heading-1>
+                {{t 'search_help.header_1'}}
+            </h2>
+            <p data-test-help-body-1>{{t 'search_help.body_1'}}</p>
+            <span local-class='pagination'>
+                <p local-class='enumeration' data-test-help-enumeration-1>{{t 'search_help.index_1'}}</p>
+                <span>
+                    <Button
+                        data-test-help-skip-1
+                        type='reset'
+                        local-class='skip-button'
+                        onclick={{fn (mut this.showTooltip1) false}}
+                    >
+                        {{t 'search_help.footer.skip'}}
+                    </Button>
+                    <Button
+                        data-test-help-next-1
+                        type='submit'
```

Comment:
I don't think these should be `type='submit'` since this isn't part of any `<form>` element, and no data is being sent

Reply:
Agreed, this is now removed.